lib/api/exception.dart

```diff
@@ -11,13 +13,26 @@ sealed class ApiRequestException implements Exception {
   /// and its route name would be "getMessages".
   final String routeName;
 
-  /// A user-facing description of the error.
+  /// Server supplied user-facing description of the error.
+  ///
+  /// For [ZulipApiException] this may be supplied by the server as the `message`
+  /// property in the JSON response.
```

Comment:
Why the change from `is` to `may be`? Seems like it still is.

Reply:
You're right, the `may be` applied to the broader `ApiRequestConnection` class, where in some cases the `message` was supplied by the server, but that is the case specifically of `ZulipApiException`